geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/GatewayCommandsUtils.java

```diff
@@ -26,4 +27,13 @@ public static void accumulateStartResult(TabularResultData resultData, String me
     resultData.accumulate("Result", Status);
     resultData.accumulate("Message", message);
   }
+
+  public static void accumulateStartResult(TabularResultModel resultData, String member,
```

Comment:
I would avoid adding on to these kind of Utilities. Our goal is to get rid of this class eventually.